docs/csharp/language-reference/keywords/out-generic-modifier.md

```diff
@@ -13,6 +13,8 @@ ms.assetid: f8c20dec-a8bc-426a-9882-4076b1db1e00
 For generic type parameters, the `out` keyword specifies that the type parameter is covariant. You can use the `out` keyword in generic interfaces and delegates.
 
 Covariance enables you to use a more derived type than that specified by the generic parameter. This allows for implicit conversion of classes that implement variant interfaces and implicit conversion of delegate types. Covariance and contravariance are supported for reference types, but they are not supported for value types.
+> Variant interface
```

Comment:
Instead of adding this definition, we recommend changing "variant interfaces" to "covariant interfaces" in the paragraph above.